react_app/src/hooks/useSearches.tsx

```diff
@@ -108,9 +113,26 @@ export function useSearch() {
     }
   }
 
-  function addSearchById(id: Search['id'], status: Search['status'] = undefined) {
-    setSearchIds([id, ...searchIds])
-    setSearchStatus(prev => ({ ...prev, id: status }))
+  async function addSearchById(id: Search['id']) {
```

Comment:
Added a call to the API checking the ID exists so as to not cause issues downstream.